media-source-respec.html

```diff
@@ -2706,6 +2714,367 @@ <h3>Event Summary</h3>
       </section>
     </section>
 
+    <section data-cite="mimesniff" data-dfn-for="ManagedMediaSource">
+      <h2><dfn>ManagedMediaSource</dfn> interface</h2>
+      <p>
+        A {{ManagedMediaSource}} is a {{MediaSource}} that actively manages its
+        memory content. Unlike a {{MediaSource}}, the [=user agent=] MAY evict it
+        from a {{ManagedSourceBuffer}} when under [=ManagedMediaSource/memory
+        pressure=].
+      </p>
+
+      <pre class="idl">
+      [Exposed=(Window,DedicatedWorker)]
+      interface ManagedMediaSource : MediaSource {
+        constructor();
+        readonly attribute boolean streaming;
+        attribute EventHandler onstartstreaming;
+        attribute EventHandler onendstreaming;
+
+        static boolean isTypeSupported(DOMString type);
```

Comment:
Does this not carry over automatically?

Reply:
It would. However being a static it requires the interface type to be defined. 
on iPhone, the MediaSource interface itself isn’t defined and you can’t override a static method. It was also considered that the range of what could be supported in Managed MSE not always be identical to MSE. 

And as such, the interface method was duplicated.

Reply:
Hmm, that seems like it just introduces more spec text than necessary. I'd probably drop this section unless you really need it then.

Reply:
I've removed the method entirely

Reply:
So the advice to developers would be to use Media Capabilities? (which sounds like a good thing). It could be worth adding an informative note to say that.

Reply:
No. You use MediaSource.isTypeSupported() from the base class.
There's no longer a distinction between what ManagedMediaSource can support and MediaSource